webapp/packages/plugin-sql-editor-new/src/SQLEditor/SQLCodeEditorPanel/useSQLCodeEditorPanel.ts

```diff
@@ -35,6 +36,9 @@ export function useSQLCodeEditorPanel(data: ISQLEditorData, editor: IEditor) {
       onQueryChange(query: string) {
         this.data.setScript(query);
       },
+      onCursorChange(begin: number, end?: number) {
+        this.data.setCursor(begin, end);
+      },
       onUpdate(update: ViewUpdate) {
         const transactions = update.transactions.filter(t => t.selection !== undefined);
         const lastTransaction = transactions[transactions.length - 1] as Transaction | undefined;
```

Comment:
seems like we can remove onUpdate completely (it's now replaced with onQUeryChange and onCursorChange